doc/modules/mod_event_pusher_push.md

```diff
@@ -0,0 +1,52 @@
+### Module Description
+This module is a backend of [mod_event_pusher] that implements [XEP-0357: Push Notifications](https://xmpp.org/extensions/xep-0357.html).
+This module implements [XEP-0357: Push Notifications](https://xmpp.org/extensions/xep-0357.html).
```

Comment:
This module is a backend of [mod_event_pusher] that implements [XEP-0357: Push Notifications](https://xmpp.org/extensions/xep-0357.html).
This module implements [XEP-0357: Push Notifications](https://xmpp.org/extensions/xep-0357.html).

information it these two lines seems duplicated